### PR TITLE
Update documentation after stat var migration

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: api/python/place_in.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Places within a Place
-nav_order: 10
+title: Places Within a Place
```
Comment (beets) [inline api/python/place_in.md:3]: please revert the capitalization of "within" for consistency (anne was careful to follow the google documentation style guide)

File: api/python/place_in.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
...
+nav_order: 6
```
Comment (beets) [inline api/python/place_in.md:4]: any reason to update the nav ordering?

File: api/python/query.md [hunk 3]
```diff
@@ -180,4 +186,4 @@ Traceback (most recent call last):
 KeyError: '?earthquake'
 ```
 
-These examples and errors, along with explanations and fixes for the errors, are available in notebook form at <https://colab.research.google.com/drive/1Jd0IDHnMdtxhsmXhL5Ib5tL0zgJud1k5?usp=sharing>.
\ No newline at end of file
+These examples and errors, along with explanations and fixes for the errors, are available in this [notebook](https://colab.research.google.com/drive/1Jd0IDHnMdtxhsmXhL5Ib5tL0zgJud1k5?usp=sharing).
```
Comment (beets) [inline api/python/query.md:189]:
```suggestion
These examples and errors, along with explanations and fixes for the errors, are available in this [Python notebook](https://colab.research.google.com/drive/1Jd0IDHnMdtxhsmXhL5Ib5tL0zgJud1k5?usp=sharing).
```

File: api/rest/place_in.md [hunk 4]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Places within a Place
-nav_order: 9
+title: Places Within a Place
```
Comment (beets) [inline api/rest/place_in.md:3]: please revert

File: api/sheets/places_in.md [hunk 5]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: default
-title: Places within a Place
-nav_order: 5
+title: Places Within a Place
```
Comment (beets) [inline api/sheets/places_in.md:3]: please revert

File: data_model.md [hunk 6]
```diff
@@ -12,8 +12,8 @@ It is important to have an underlying model capable of expressing the breadth of
...
+1. The data model for the base layer is the one used by [Schema.org](https://schema.org). This models the world as a set of entities, with attributes and relationships between entities. There is a taxonomy of entities and each entity is an instance of at least one of the types in the taxonomy. The types and relations types are also entities. This kind of structure has its origins in knowledge representation systems such as KRL and Cyc, and has recently found adoption under the name "knowledge graph". The [Node and SPARQL APIs](/api) provide access to this view. The [Data Commons Graph Browser](https://datacommons.org/browser) allows one to browse through Data Commons in this raw graph view
+1. Time series view provides a set of time series for combinations of entities and variables ([Statistical Varable](https://datacommons.org/browser/StatisticalVariable), in Data Commons parlance). The [DCGET API](/api/sheets/get_variable.html) provides API access to this view of the data and the [Data Commons Timeline Tool](https://datacommons.org/tools/timeline) allows one to browse Data Commons in this view.
```
Comment (beets) [inline data_model.md:16]:
```suggestion
1. Time series view provides a set of time series for combinations of entities and variables ([Statistical Variable](https://datacommons.org/browser/StatisticalVariable), in Data Commons parlance). The [DCGET API](/api/sheets/get_variable.html) provides API access to this view of the data and the [Data Commons Timeline Tool](https://datacommons.org/tools/timeline) allows one to browse Data Commons in this view.
```

File: glossary.md [hunk 7]
```diff
@@ -10,51 +10,49 @@ This page contains a selection of key terms important to understanding the struc
...
 
 Property of statistical variables that measure proportions, used in conjunction with the measurementDenominator property to indicate the multiplication factor applied to the proportion's denominator (with the measurement value as the final result of the multiplication) when the numerator and denominator are not equal.
 
-As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/2mthzyv99kd73). Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
+As an example, in 1999, [approximately 36% of Canadians were Internet users](https://datacommons.org/browser/dc/o/0d9e3dd3y6yt3). Here the measured value of `Count_Person_IsInternetUser_PerCapita` is 36, and the scaling factor or denominator for this per capita measurement is 100. Without the scaling factor, we would interpret the value to be 36/1, or 3600%.
 
-A complete list of properties can be found at <https://datacommons.org/browser/scalingFactor>.
```
Comment (beets) [inline glossary.md:47]: please keep this line, even if redundant. the headings don't look very clickable at the moment.

File: glossary.md [hunk 8]
```diff
@@ -66,12 +64,13 @@ Given tabular data such as the following:
...
 USA -- name --------------> United States of America
 USA -- containedInPlace --> northamerica
 ```
 
-- **[Unit](https://datacommons.org/browser/unit)**
+### **[Unit](https://datacommons.org/browser/unit)**
 
-The unit of measurement. Examples include [kilowatt hours](https://datacommons.org/browser/KilowattHour), [inches](https://datacommons.org/browser/Inch), and [Indian Rupees](https://datacommons.org/browser/IndianRupee). A complete list of properties can be found at <https://datacommons.org/browser/unit>.
\ No newline at end of file
+The unit of measurement. Examples include [kilowatt hours](https://datacommons.org/browser/KilowattHour), [inches](https://datacommons.org/browser/Inch), and [Indian Rupees](https://datacommons.org/browser/IndianRupee). A complete list of properties can be found in the [browser](https://datacommons.org/browser/unit).
```
Comment (beets) [inline glossary.md:76]:
```suggestion
The unit of measurement. Examples include [kilowatt hours](https://datacommons.org/browser/KilowattHour), [inches](https://datacommons.org/browser/Inch), and [Indian Rupees](https://datacommons.org/browser/IndianRupee). A complete list of properties can be found in the [graph browser](https://datacommons.org/browser/unit).
```

File: glossary.md [hunk 9]
```diff
@@ -10,51 +10,49 @@ This page contains a selection of key terms important to understanding the struc
 
 ## Term List
 
-- **[Cohort](http://browser.datacommons.org/kg?dcid=CohortSet)**
+### **[Cohort](https://datacommons.org/browser/CohortSet)**
```
Comment (beets) [inline glossary.md:13]: the bold might not be necessary if it is now also a heading.

Comment (beets) [reply glossary.md:13]: should we make the links clearer by making it separate from the heading? e.g.

### Cohort ([graph node](https://datacommons.org/browser/CohortSet))

or

### Cohort 
 ([see the browser node](https://datacommons.org/browser/CohortSet))

**Review by beets — COMMENTED** [review]
thanks for all these fixes!

could you update your editor settings to only update lint / whitespace in files with modifications?

please also note in the PR description that some obsolete pages are deleted

File: glossary.md (hunk 9 above)
Comment (shifucun) [reply glossary.md]: Good idea. Updated

File: glossary.md (hunk 7 above)
Comment (shifucun) [reply glossary.md:47]: Added this back to be consistent with other items

File: api/python/place_in.md (hunk 2 above)
Comment (beets) [reply api/python/place_in.md:4]: gotcha, sg

**Review by beets — APPROVED** [review]
thanks for the updates!

**Comment by shifucun** [comment]
Thanks for review!

**Pull request merged**